cip-drep-metadata/README.md

```diff
@@ -0,0 +1,231 @@
+---
+CIP: ?
+Title: DRep Metadata
+Category: ?
+Status: Proposed
+Authors:
+    - Thomas Upfield <thomas.upfield@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-02-07
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta     | For meta-CIPs which typically serves another category or group of categories.
+- Wallets  | For standardisation across wallets (hardware, full-node or light).
+- Tokens   | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata | For proposals around metadata (on-chain or off-chain).
+- Tools    | A broad category for ecosystem tools not falling into any other category.
+- Plutus   | Changes or additions to Plutus
+- Ledger   | For proposals regarding the Cardano ledger (including Reward Sharing Schemes)
+- Catalyst | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
```

Comment:
Please remove all this comment scaffolding from the CIP template.

Reply:
Looks all gone now (in a later commit not linked back here).

Reply:
done. thx.